packages/solid/src/index.tsx

```diff
@@ -0,0 +1,4 @@
+
+export * from './components/Auth/index'
+export * from './../common/theming/defaultThemes'
+export * from './../common/lib/Localization/index'
```

Comment:
This path isn't resolving as the directory name is lowercase (`localization`) while you are referencing capital case here (`Localization`).